endtoendtests/Azure.Functions.Java.Tests.E2E/Azure.Functions.Java.Tests.E2E/Helpers/StorageHelpers.cs

```diff
@@ -135,13 +135,15 @@ private static async Task ClearBlobContainer(string containerName)
             BlobContinuationToken blobContinuationToken = null;
             do
             {
+                if (!await cloudBlobContainer.ExistsAsync()) { continue; }
```

Comment:
Consider using `Utilities.RetryAsync` instead. like in https://github.com/Azure/azure-functions-java-worker/blob/62d97d57b2f7efe9820eecb93c5a5aa7ff2dc4e2/endtoendtests/Azure.Functions.Java.Tests.E2E/Azure.Functions.Java.Tests.E2E/Helpers/StorageHelpers.cs#L111